packages/core-contracts/contracts/ownership/OwnableMixin.sol

```diff
@@ -4,24 +4,38 @@ pragma solidity ^0.8.0;
 abstract contract OwnableMixin {
     error OnlyOwnerAllowed();
 
+    struct OwnerNamespace {
+        address owner;
+        address nominatedOwner;
+    }
+
     modifier onlyOwner() {
         _onlyOwner();
         _;
     }
 
+    modifier onlyOwnerIfSet() {
+        address owner = _getOwner();
+        if (owner != address(0) && msg.sender != owner) {
```

Comment:
Here you can use the _onlyOwner() and reduce it to follow the same mechanics than onlyOwner().

i.e. 
```
if (owner != address(0)) {
  _onlyOwner();
}
_;  
```

Reply:
good suggestion, implemented!